plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js

```diff
@@ -832,11 +832,11 @@ function generateText(taskNode) {
 function getArgumentName(nameValue) {
     if (nameValue) {
         if (nameValue.indexOf("Role") != -1)
-            return "Role";
+            return "role";
         else if (nameValue.indexOf("Literal") != -1)
-            return "Literal";
+            return "literal";
         else if (nameValue.indexOf("Expression") != -1)
-            return "Expression";
+            return "expression";
```

Comment:
Thanks. These strings were moved to constants in https://github.com/wso2/devstudio-tooling-bps/pull/30